site/content/agent-app-widgets/sample-apps.md

```diff
@@ -3,13 +3,42 @@ weight: 20
 ---
 
 
-# Sample Widgets
+# Sample Apps
 
-We've prepared a repository of sample apps for your convenience.
+## Agent App Widgets
 
-* [Tag Master](https://github.com/livechat/sample-apps/tree/master/tag-master): widget allows user to create, view and delete tags and canned responses in easy and predictable way.
-* [Progress](https://github.com/livechat/sample-apps/tree/master/progress): a simple app that creates reports based on data from chats.
-* [Supervisor](https://github.com/livechat/sample-apps/tree/master/supervisor): simple widget for helping you to monitor weekly progress of your agents and also their availability.
+If you want to experiment with some of the already existing apps, make sure to check out the repository we've prepared. Once installed, these widgets will be displayed in the right sidebar of the Agent App.
 
+* [iFrame loader](https://glitch.com/edit/#!/livechat-load-iframe?path=README.md:1:0): allows to embed any website.
+* [Visitor preview widget](https://glitch.com/edit/#!/livechat-sample-agent-app): displays data of the currently selected visitor.
+* [Tag Master](https://github.com/livechat/sample-apps/tree/master/tag-master): allows user to create, view, and delete tags and canned responses in easy and predictable way.
+* [Progress](https://github.com/livechat/sample-apps/tree/master/progress): creates reports based on data from chats.
+* [Supervisor](https://github.com/livechat/sample-apps/tree/master/supervisor): helps to monitor the weekly progress of your agents, as well as their availability.
 
-<a href="https://github.com/livechat/sample-apps/" class="cta green" target="_blank"><img src="../assets/images/github-logo.svg" style="background: none;margin-right:5px;" width="20"/> See the repo</a>
\ No newline at end of file
+
+<a href="https://github.com/livechat/sample-apps/" class="cta green" target="_blank"><img src="../assets/images/github-logo.svg" style="background: none;margin-right:5px;" width="20"/> See the repo</a>
+
+
+<!-- Fullscreen Apps -->
+
+## Fullscreen Apps
+
+### Overview
+
+This functionality gives developers a new placement for the applications they create. Their apps can be displayed fullscreen, right in the Agent App. The application icon is localized in the **navigation bar**. Clicking the icon loads the app URL as an iframe.
+
+        ---- GRAPHIC HERE (navigation icon) ----
+         ----- GRAPHIC HERE (fullscreen app)----
+
+Keep in mind that this feature is not supported on the mobile version of the **LiveChat Agent App**.
+
+### Benefits
+
+The greatest advantage of Fullscreen Applications is _content switching_. Users no longer need to switch between Agent App and tabs in the browser. As long as the application they work with is installed as a Fullscreen App, they can perform all the operations without leaving the Agent App.
```

Comment:
Regarding `content switching` - how about replacing it with `context switching`?

Reply:
done